### ISSUE TITLE
Photo Ordering in Albums

### ISSUE THREAD

**Issue by Bjoern3003** [PHPfox-Official/phpfox-v4-issues#2592]
There is an ordering col in database table phpfox_photo. But the Photo App does it not consider.

For example in Album Controller Line 93
`list($iCnt, $aPhotos) = Phpfox::getService('photo')->get($aConditions, 'p.photo_id DESC', $iPage, $iPageSize);`

The Default Ordering is photo_id Desc. Also in the get Function from Photo Class.

**Comment by harrison05** [comment]
I don't understand. What do you mean about this code?

Regards

**Comment by Bjoern3003** [comment]
This is an example, that the photos in Albums are orderd by id, and not by the ordering field.

**Comment by harrison05** [comment]
Currently, we don't support re-ordering Photos in a Album, so we order them by id.

**Comment by harrison05** [comment]
Currently, we don't support re-ordering Photos in an Album, so we order them by id.

**Comment by Bjoern3003** [comment]
Yes, i know it. But you can add both fields to order.

For example
Order by p.ordering ASC, p.photo_id DESC.

So you Support both variants.

**Comment by harrison05** [comment]
I understand your request. We will support this later because this is useless with the current version.

Regards.

**Comment by Bjoern3003** [comment]
But it could be useful for third Party Apps, that is working with your photo App Database and support ordering. It could be nice, If you are adding this small modification to your App, so we can use the ordering database field in your Photo App.

**Comment by harrison05** [comment]
If third Party App working on it, they can call this function and add the ordering field to sort parameter:
```php
Phpfox::getService('photo')->get($aConditions, 'p.ordering ASC', $iPage, $iPageSize);
```
Are you using our AlbumController, so you can't change source code on it?

**Comment by Bjoern3003** [comment]
Hi,

yes, we can do it in our App, but problem is, we are also using Third Party App **Photo Viewer Popup [V4] - YouNetCo** and this App is also using core Ordering. 

If we are changing it in our App, than Display Ordering in our App where good, but in Photo Viewer is the ordering as default :-\

**Comment by harrison05** [comment]
Ok, we will update to support 3rd and release in the next version of Photo App.
```php
list($iCnt, $aPhotos) = Phpfox::getService('photo')->get($aConditions, 'p.ordering ASC, p.photo_id DESC', $iPage, $iPageSize);
```

**Comment by david-phpFox** [comment]
Fixed in Core 4.7.5
